rustls-cert-gen/Cargo.toml

```diff
@@ -0,0 +1,9 @@
+[package]
```

Comment:
WDYT about adding a `description`? Did you want to wait for reworking the tool to add some of that type of metadata?

Reply:
The idea was just to add minimal metadata mainly just to test expectations about workspace functionality. But having a description doesn't hurt anything, so I added one in latest commit.